CHANGELOG.md

```diff
@@ -2,10 +2,9 @@
 
 ## Next Version
 
-#### Added
```

Comment:
Was removing this heading an accident? Let's keep it around. It's fine to remove the blank line

Reply:
@yonaskolb that was a merge error, should be fixed now.